### DISCUSSION TITLE
Unsorted Classic Site Tree Navigation

### DISCUSSION THREAD

**Discussion by Penguinlay** [requarks/wiki#3325] [Error / Bug Report]
**Describe the bug**
A clear and concise description of what the bug is:
The Classic Site Tree Navigation menu is not sorted.

**To Reproduce**
1. Go to / Select Classic Site Tree as the navigation mode.
2. Create a few pages if you have not already.
3. You will see that the pages in Classic Site Tree navigation mode are not sorted.
4. There is no built-in options to sort them in admin setting (without programmatically editing UI or injecting custom JS, of course)

**Expected behavior**
Classic Site Tree Navigation should be sorted?

**Screenshots**

**Host Info (please complete the following information):**
 - OS: Ubuntu 20.10; Docker Version 20.10.3, Build 48d30b5
 - Wiki.js version: 2.5.170
 - Database engine: PostgreSQL: 11.10, Node.js: 14.14.0

**Additional context**

**Comment by FabriceSalvaire** [comment]
Now it is sorted, but by category (first directories and pages).
